docs/cli.md

```diff
@@ -0,0 +1,120 @@
+# The CLI commands of pixi
+With `pixi` you can install packages in global space or local to the environment in a project.
+
+## Pixi Project Commands
+
+|  Command | Use case  |
+|--|--|
```

Comment:
Could be nice to make this look good in Markdown as well (there are some markdown prettier / markdown table plugins for VSCode).

Reply:
Done!